screens/TimerScreen.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { Image, StyleSheet, ScrollView, TouchableOpacity, View } from 'react-native';
+import { Input } from 'react-native-elements';
+import { Keyboard } from 'react-native';
+
+import TabBarIcon from '../components/TabBarIcon';
+import { MonoText } from '../components/StyledText'
+
+const timerIcon = require('../assets/images/hourglass.png')
+
+export default function TimerScreen() {
+  const [newTaskName, setNewTaskName] = useState('');
+  const [tasks, setTasks] = useState([]);
+  const [timerAmount, setTimerAmount] = useState(FIVE);
+
+  const TWENTY_FIVE = 25 * 60;
+  const FIFTEEN = 15 * 60;
+  const FIVE = 5 * 60;
+
+  const timeHitSlop = {top: 10, bottom:10, left: 10, right:10};
+
+    const handleTaskName = (inputValue) => {
+    console.log('Name ', inputValue);
+    setNewTaskName(inputValue);
+  };
+
+  const onAddTask = () => {
+    //Why I don't have to import Keyboard from react native ?
+    Keyboard.dismiss()
+
+    if (newTaskName !== '') {
+      const updateTasks = [newTaskName, ...tasks];
+      setTasks(updateTasks);
+      setNewTaskName('');
+    } else {
+      console.log('Invalid task name, it must not be empty');
+    }
+  }
+
+  //onSubmitEditing = if "enter" is pressed on the keyboard, onAddTask is fired
+  return (
+
+  <View >
+    <View style={styles.titleContainer}>
+      <MonoText style={styles.title} > Timer </MonoText>
+      <Image style={styles.timerIcon} source={timerIcon}></Image>
+    </ View >
+
+    <View style={styles.inputContainer}>
+        <Input placeholder='Type of task' value={newTaskName} onChangeText={handleTaskName}
+          onSubmitEditing={onAddTask}/>
+        <TouchableOpacity onPress={onAddTask}>
+        <TabBarIcon name="md-add" />
+      </TouchableOpacity>
+    </View >
+
+      <View style={styles.minutesContainer} >
+        {
+          [FIVE, FIFTEEN, TWENTY_FIVE].map(time =>(
+            <TouchableOpacity key={time} hitSlop={timeHitSlop}
+              onPress={(time) => setTimerAmount({ timerAmount : time})}>
+              <MonoText style={[styles.minutes,
+                { color: time ==! undefined ? '#dcdf4a' : '#4ba9bc'  }]}> { time / 60}
+              </MonoText >
+            </TouchableOpacity>
+          ))
+        }
+     </View >
+
```

Comment:
@donni106 Here I'm trying to set to state the selected time, and change the colour when user tab on one of the this time options . It is not working , where do you think am I mistaking?

Reply:
at first sight `time ==! undefined` this operation is not existing. perhaps `!==` is what you want.
reading the code was not as easy as expected, because it seems that you did not care much about linting?

Reply:
Looks so much better now ! 
<img width="1073" alt="Screenshot 2021-05-20 at 16 03 38" src="https://user-images.githubusercontent.com/43641321/118992464-f5777380-b984-11eb-8bbd-d9281bc5fbab.png">